app/controllers/posts_controller.rb

```diff
@@ -27,6 +27,13 @@ def create
     end
   end
 
+  def destroy
+    @post = Post.find(params[:id])
+    @post.destroy
+
+    redirect_to user_path(params[:user_id]), status: :see_other
+  end
+
```

Comment:
- [ ] Please make sure to add the authorization here, on your posts controller. You should add `load_and_authorize_resource` at the top of the controller in order to make sure that `CanCanCan` gem works properly.

Reply:
@Lordkaito I did the change as requested. Thank you for improving our App